superagi/jobs/agent_wait_step_executor.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+
+from sqlalchemy.orm import sessionmaker
+
+from superagi.agent.agent_workflow_step_wait_handler import AgentWaitStepHandler
+from superagi.models.agent_execution import AgentExecution
+from superagi.models.db import connect_db
+from superagi.models.workflows.agent_workflow_step import AgentWorkflowStep
+from superagi.models.workflows.agent_workflow_step_wait import AgentWorkflowStepWait
+from superagi.worker import execute_agent
+
+engine = connect_db()
+Session = sessionmaker(bind=engine)
+
+
+class AgentWorkflowStepWaitExecutor:
+
+    def execute_waiting_workflows(self):
```

Comment:
add this in agent_executor itself. We dont need another executor just to manage wait steps.

Reply:
Yes putting it in agent_executor